src/parsers/component.cr

```diff
@@ -7,7 +7,7 @@ module Mint
 
     def component : Ast::Component | Nil
       start do |start_position|
-        comment = self.comment
+        comment = self.comment || self.inline_comment
```

Comment:
There are a lot of places where the `comment` function is called. Adding `|| self.inline_comment` to each would not be a good solution.

Probably we should:
- rename the `comment` function to `block_comment`
- create a new `comment` function which parses a `block_comment` or an `inline_comment`
- rename the `Ast::Comment` node to `Ast::BlockComment` and create an alias: `alias Comment = BlockComment | InlineComment`

Then all current related code would work unchanged.